configuration/editors.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: documentation
+title: Editors
+---
+
+{% include base.html %}
+
+# Editors - Different ways to power up your textual configuration
+
+There are several existing solutions, that can help you configuring your openHAB instance in a textual way.
+Mainly there are two bigger projects for this. 
+_Eclipse Smart Home - Designer_ as a standalone editor and _openhab-vscode_ as an extension for the Visual Studio Code open source editor.
+Additionally there some syntax highligting solutions existing for other text editors.
+This documentation page can give you some guidance in choosing the right one for you and setting it up.
+
+{::options toc_levels="2"/}
+
+* TOC
+{:toc}
+
+## openhab-vscode
+
+openhab-vscode is an extension for the [Visual Studio Code](https://code.visualstudio.com) editor.
+
+### Manual Installation
+
+1. Download the latest release from <https://github.com/openhab/openhab-vscode/releases>
+2. Open command line in the same folder you downloaded the extension
+3. Type the following command:
+    ```
+    code --install-extension openhab-0.0.2.vsix
+    ```
+Currently openhab-vscode isn't yet listed in [Microsoft Visual Studio Marketplace](https://marketplace.visualstudio.com/vscode), but the contributors are working on it already.
+
+[Visit the Extensions GitHub Page for further Informations](https://github.com/openhab/openhab-vscode/blob/master/README.md "GitHub Repo for the VS Code Extension")
```

Comment:
@kubawolanin how about this alternative way?
https://community.openhab.org/t/openhab-vs-code-extension-0-1-0/31202/5?u=thomdietrich